Tests/TestingApps/PurchaseTesterSwiftUI/Shared/Views/LocalReceiptView.swift

```diff
@@ -0,0 +1,251 @@
+//
+//  LocalReceiptView.swift
+//  PurchaseTester
+//
+//  Created by Nacho Soto on 1/11/23.
+//
+
+import Foundation
+import SwiftUI
+import RevenueCat
+import ReceiptParser
+
+private typealias AppleReceipt = ReceiptParser.AppleReceipt
+
+@MainActor
+struct LocalReceiptView: View {
+
+    @State
+    private var receipt: Result<AppleReceipt, Error>?
+
+    var body: some View {
+        VStack {
+            if #available(iOS 16.0, macCatalyst 16.0, macOS 13.0, *) {
+                self.form
+                    .scrollContentBackground(.hidden)
+            } else {
+                self.form
+            }
+
+            Spacer(minLength: 10)
+
+            Button {
+                Purchases.shared.restorePurchases { _, error in
+                    if let error = error {
+                        self.receipt = .failure(error)
+                    } else {
+                        Task<Void, Never> {
+                            await self.refreshReceipt()
+                        }
+                    }
+                }
+            } label: {
+                Text("Refresh receipt")
+            }
+            .buttonStyle(.borderedProminent)
+        }
+        .navigationTitle("Local Receipt")
+        .task {
+            await self.refreshReceipt()
```

Comment:
do we need this to happen automatically? If we're doing this when we load the view, then I think the refresh button will be mostly irrelevant, since you'll only be able to modify the receipt by either waiting for a subs to expire or going through manage subs in App Store or Xcode. 

I'd vote for this to be manual just so that the debugging tool doesn't modify the environment unless you ask it to, but I'm more than open to feedback on this

Reply:
This is probably a bad name. `refreshReceipt` simply loads it, but the button calls `restorePurchases` first to use `SKReceiptRefreshRequest` before loading it again. Does that make sense?